haystack/document_stores/deepsetcloud.py

```diff
@@ -436,6 +436,24 @@ def query(
         docs = [Document.from_dict(doc) for doc in doc_dicts]
         return docs
 
+    def query_batch(
+        self,
+        queries: Union[str, List[str]],
+        filters: Optional[
+            Union[
+                Dict[str, Union[Dict, List, str, int, float, bool]],
+                List[Dict[str, Union[Dict, List, str, int, float, bool]]],
+            ]
+        ] = None,
+        top_k: int = 10,
+        custom_query: Optional[str] = None,
+        index: Optional[str] = None,
+        headers: Optional[Dict[str, str]] = None,
+        all_terms_must_match: bool = False,
+        scale_score: bool = True,
+    ) -> Union[List[Document], List[List[Document]]]:
+        raise NotImplementedError("DeepsetCloudDocumentStore currently does not support query_batch method.")
```

Comment:
Could we maybe add this without much overhead before the next release? Is there an issue already? It would be a shame if we can't use all the batch processing in dc because of that. What we need for that is a `self.client.query_batch()` call here and then also a `query_batch()` implementation in `IndexClient`.

Reply:
I would suggest to just call `query` here multiple times, as there is no batch_query endpoint in dc (yet). (In `query_batch()`in `IndexClient` we would need to call the `documents-query` endpoint multiple times.) Once dc has a batch_query endpoint, we can have a `query_batch()` implementation for the `IndexClient`. What do you think?

Reply:
Yes, agreed. 👍